expat.spec

```diff
@@ -1,6 +1,9 @@
-### RPM external expat 2.1.0
+### RPM external expat 2.4.6
 ## INITENV +PATH LD_LIBRARY_PATH %{i}/lib64
-Source: http://downloads.sourceforge.net/project/%{n}/%{n}/%{realversion}/%{n}-%{realversion}.tar.gz
+%define realversion %(echo %v | sed -e 's/\./_/g')
```

Comment:
realversion is already sed by our build-system please use some other variable e.g. download_verison